static/app/components/globalSelectionLink.tsx

```diff
@@ -7,6 +6,7 @@ import Link from 'sentry/components/links/link';
 import {extractSelectionParameters} from 'sentry/components/organizations/globalSelectionHeader/utils';
 
 type Props = WithRouterProps & {
+  children?: React.ReactNode;
```

Comment:
since this prop is optional, can we move it to the bottom of the object? :)

Reply:
will move it in a follow up!

Reply:
https://github.com/getsentry/sentry/pull/30711